app/Services/BulkService.php

```diff
@@ -405,11 +405,14 @@ protected function updateRelations(array $filledFields)
             if ($relation->owner_id == Arr::get($filledFields, 'target_id') || ($relation->target_id == Arr::get($filledFields, 'owner_id'))) {
                 continue;
             }
-
+            if (request()->update_mirrored && $relation->mirror) {
+                $relation->mirror->update(\array_diff_key($filledFields, ["target_id" => "xy", "owner_id" => "xy"]));
```

Comment:
This is a lot of code on a single line, and not sure why VSCode added a \ in front of array_diff_key, which is a default php function.

Cleaner would be 
> use Illuminate\Support\Arr;
> ....
> $mirrorFields = Arr::except($filledFields, ['target_id', 'owner_id']);
> $relation->mirror->update($mirrorFields);

More info on https://laravel.com/docs/9.x/helpers#method-array-except